UI/frontend-plugins/decklink-output-ui/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+project(decklink-output-ui)
+
+if (APPLE)
```

Comment:
Style:  It seems like the preferred style in CMake is `if(<condition>)`.  Remove the space after `if`.